documentation/docs/configuration/projects.md

```diff
@@ -2,22 +2,42 @@
 sidebar_position: 3
 ---
 
-# Configuration Structure
+# Projects
 
-Configuration files are ordered by `project` in the projects folder. Project folder can only contain:
+A project is a folder containing, which contains specially named
+sub-folders, representing APIs. This API folders then contain another
+layer of folders defining confiugrations. Then finally, this configuration
```

Comment:
so it's now project-folder -> api -> ???folder -> config ? 
If yes, is that middle layer mandatory? the sentences makes it sound like it is